src/components/nav_bar.jsx

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 const NavBar = () => (
-  <div>
-    <Link to='/career'>career</Link>
-    <Link to='/list'>list</Link>
-    <Link to='/information'>info</Link>
+  <div className='navBar'>
```

Comment:
https://i0.wp.com/www.cheenachatti.com/engine/uploads/2013/04/mutton-kabab.jpg?fit=700%2C371

Reply:
confused @Karyum  — navBar and imgContainer are kebab aren't they?

Reply:
ahhhhh - i see. kebab-case . Don't worry about this. my mistake